api/guides/filtering.md

```diff
@@ -4,31 +4,79 @@ title: Filtering
 
 # Filtering
 
-At times it's desirable to request a subset of records by means of a filter. Some API endpoints support the ability to filter by specific attributes. Filterable attributes will be noted on their respective index endpoint documentation areas.
+When using listing (GET) endpoints in the Adobe Experience Platform Launch API, you may find it necessary to limit the returned results to a subset of records. To accomplish this, many of the API's listing endpoints support the ability to filter by specific attributes.
```

Comment:
It's a nitpick, but can we call them "list" endpoints rather than "listing endpoints".